tasks/main.yml

```diff
@@ -15,33 +15,19 @@
     - ansible_distribution == "Ubuntu"
     - ansible_distribution_major_version|int < 20
 
-- name: Add unstable repository (Debian)
+- name: Add backports repository (Debian)
   copy:
-    src: templates/unstable.list
-    dest: /etc/apt/sources.list.d/unstable.list
+    src: templates/backports.list
+    dest: /etc/apt/sources.list.d/backport.list
   when:
     - ansible_distribution == "Debian" and debian_enable_testing
 
-- name: Pin packages (Debian)
-  copy:
-    src: templates/limit-unstable
-    dest: /etc/apt/preferences.d/limit-unstable
-  when:
-    - ansible_distribution == "Debian" and debian_pin_packages
-
-- name: Get kernel version (Debian)
-  command: uname -r
-  register: kernel
-  when:
-    - ansible_distribution == "Debian"
-
-- name: Install kernel headers (Debian)
-  apt:
-    state: present
-    update_cache: true
-    name: linux-headers-{{ kernel.stdout }}
-  when:
-    - ansible_distribution == "Debian"
+    # - name: Pin packages (Debian)
```

Comment:
It seems like the pinning config is unnecessary nowadays, I think this section can be removed entirely

Reply:
I just removed it